test/controllers/manageBusinesses/cease/IncomeSourceEndDateControllerSpec.scala

```diff
@@ -213,6 +213,80 @@ class IncomeSourceEndDateControllerSpec extends MockAuthActions with MockSession
                 redirectLocation(result) shouldBe Some(redirectLocationResult)
               }
             }
+            "display before tax year error" when {
+              "form is completed successfully" in {
+                setupMockSuccess(mtdRole)
+                enable(IncomeSourcesFs)
```

Comment:
this should be IncomeSourceNewFs i.e., new journey but this issue is existing from before and I'll fix them all in my PR.